packages/expo/src/notifier.js

```diff
@@ -15,6 +16,18 @@ module.exports = (opts) => {
   // handle very simple use case where user supplies just the api key as a string
   if (typeof opts === 'string') opts = { apiKey: opts }
 
+  // ensure opts is actually an object (at this point it
+  // could be null, undefined, a number, a boolean etc.)
+  opts = { ...opts }
+
+  // attempt to fetch apiKey from app.json if we didn't get one explicitly passed
+  if (!opts.apiKey &&
+    Constants.manifest.extra &&
```

Comment:
this may be overly paranoid, but can we trust that Constants.manifest will be non-null?

Reply:
An expo app should always have a `app.json` manifest, but I can't see it hurting having a null check